src/defs/clientProfilesRepresentation.ts

```diff
@@ -3,7 +3,8 @@ import ClientProfileRepresentation from './clientProfileRepresentation';
 /**
  * https://www.keycloak.org/docs-api/15.0/rest-api/#_clientprofilesrepresentation
  */
+
 export default interface ClientProfilesRepresentation {
   globalProfiles?: ClientProfileRepresentation[];
   profiles?: ClientProfileRepresentation[];
-}
\ No newline at end of file
+}
```

Comment:
Hmmm, perhaps we should add something to enforce this last line in code-style. Out of scope for this PR though.